lab_1/main.py

```diff
@@ -140,19 +135,21 @@ def detect_language(unknown_profile: dict, profile_1: dict,
     :param top_n: a number of the most common words
     :return: a language
     """
-    try:
-        p1_cross = compare_profiles(unknown_profile, profile_1, top_n)
-        p2_cross = compare_profiles(unknown_profile, profile_2, top_n)
+    # the check are divided in order to avoid long lines
+    if not isinstance(unknown_profile, dict) or not isinstance(profile_1, dict):
+        return None
+    if not isinstance(profile_2, dict) or not isinstance(top_n, int):
+        return None
 
-        if p1_cross > p2_cross:
-            return profile_1["name"]
-        if p2_cross > p1_cross:
-            return profile_2["name"]
-        alphabetical = sorted(list[profile_1["name"], profile_2["name"]])
-        return alphabetical[0]
+    p1_cross = compare_profiles(unknown_profile, profile_1, top_n)
```

Comment:
better to name like first_cross
numbers are not commonly used in names

Reply:
Will change them to remember it.